apps/shipments/models.py

```diff
@@ -341,3 +343,98 @@ def update_vault_hash(self, vault_hash, rate_limit=False):
     VALID_UNTIL = 24
     FUNDING_TYPE = FundingType.SHIP.value
     SHIPMENT_AMOUNT = 1
+
+
+class TrackingData(models.Model):
+    id = models.CharField(primary_key=True, default=random_id, max_length=36)
+    created_at = models.DateTimeField(auto_now_add=True)
+    device_id = models.CharField(blank=True, null=True, max_length=36)
+    shipment = models.ForeignKey(Shipment, on_delete=models.PROTECT, null=True)
+    latitude = models.FloatField(default=None)
+    longitude = models.FloatField(default=None)
+    altitude = models.FloatField(default=None)
+    source = models.CharField(max_length=10)
+    uncertainty = models.IntegerField(blank=True, null=True)
+    speed = models.IntegerField(blank=True, null=True)
+    timestamp = models.CharField(max_length=50)
```

Comment:
Sorry I'm just now catching this. Can you please change timestamp to be a DateTimeField? And in the serializer you can convert it to/from ISO-8601 format. This will help us with the sorting/filtering of TrackingData. Also then we don't need the datetime_timestamp() method.